test/namespace_admin_test.py

```diff
@@ -86,18 +77,28 @@ def setup_service_accounts(self):
         self.wait_for_predicate(self.service_accounts_are_validated)
         self.log.info("Service accounts have been set up and validated")
 
-    def deploy_network_in_team_namespaces(self):
-        self.log.info("Deploy networks to team namespaces")
-        self.log.info(self.warnet(f"deploy {self.network_dir} --to-all-users"))
+    def setup_network(self):
+        if self.current_context == self.bob_context:
+            self.log.info(f"Allowing {self.current_context} to update the network...")
+            assert self.this_is_the_current_context(self.bob_context)
+            self.warnet(f"deploy {self.network_dir}")
+        else:
+            self.log.info("Deploy networks to team namespaces")
+            assert self.this_is_the_current_context(self.initial_context)
+            self.log.info(self.warnet(f"deploy {self.network_dir} --to-all-users"))
```

Comment:
I think I had a lower tank count in an earlier version of this test; however, since I decided to extend the Scenarios test, it meant that I would need to use whatever it is using.

Reply:
I suppose I could override network in my namespace_admin test. That would let us use fewer nodes if we wanted to.